packages/voice/src/app/agent/chat.tsx

```diff
@@ -675,8 +675,10 @@ export class WebRtcChatManager implements ChatManager {
     const audioTrack = track as RemoteAudioTrack;
     audioTrack.on(TrackEvent.AudioPlaybackStarted, () => console.log(`[chat] audio playback started`));
     audioTrack.on(TrackEvent.AudioPlaybackFailed, (err) => console.error(`[chat] audio playback failed`, err));
+    audioTrack.setAudioContext(this.audioContext);
```

Comment:
OOC, why is this needed?

Reply:
The real answer is I was just playing with it and saw the function exists, thought maybe I should connect the two objects because they seemed to be coming from different places ... so basically just trial and error 😅

Reply:
OK... would be good to try to understand what is happening internally and add a comment indicating that as it's not clear at all to me how setting this context affects our own visualizer objects

Reply:
Sorry I didn't get to figure it out. I left a TODO for when I come back to it.